src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,119 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import PropTypes from 'prop-types';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import getMovieFromServer from '../../api';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ addMovie, movies }) => {
+  const [movie, setMovie] = useState(null);
+  const [Title, setTitle] = useState('');
+  const [hasMovie, setHasMovie] = useState(false);
+  const [cantFind, setCantFind] = useState(false);
+
+  const handleFindMovie = async() => {
+    const newMovie = await getMovieFromServer(Title);
+
+    // eslint-disable-next-line
+    if (newMovie.hasOwnProperty('Error')) {
+      setHasMovie(false);
+      setCantFind(true);
+
+      return;
+    }
+
+    setCantFind(false);
+    setHasMovie(true);
+    setMovie(newMovie);
+  };
+
+  const handleAddMovie = () => {
+    if (!movie) {
+      return;
+    }
+
+    if (movies.some(film => film.imdbID === movie.imdbID)) {
+      return;
+    }
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+    addMovie(movie);
+    setTitle('');
+    setMovie(null);
+    setHasMovie(false);
+  };
+
+  return (
+    <>
+      <form
+        className="find-movie"
+        onSubmit={(event) => {
+          event.preventDefault();
+        }}
+      >
+        <div className="field">
+          <label className="label" htmlFor="movie-Title">
+            Movie Title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-Title"
+              placeholder="Enter a Title to search"
+              className={`input ${cantFind && 'is-danger'}`}
+              value={Title}
+              onChange={(event) => {
+                setTitle(event.target.value);
```

Comment:
As task description says:

> Hide the error after changing a title.

Let's handle this case also. The error should disappear after changing the input